packages/@lwc/engine/src/framework/local-registry.ts

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
Is it possible to rename this module to something else, it's not representative of what the code is doing? What about `element-reaction.ts`?

Reply:
yeah, maybe `upgradable-element.ts`